msproteomicstoolslib/data_structures/modifications.py

```diff
@@ -43,7 +44,7 @@
 import csv
 import ast
 import re
-import sys
```

Comment:
I guess we still need `sys`, see https://travis-ci.org/msproteomicstools/msproteomicstools/builds/57657023